### ISSUE TITLE
Unable to enter login/pin on Windows 10 unlock login screen after screen lock due to inactivity

### ISSUE THREAD

**Issue by Alden20** [FreeRDP/FreeRDP#7799]
Hi,

After the remote Windows 10 machine is locked for a period of inactivity, it is not possible to unlock it except by closing the connection and reconnecting again because the unlock page does not show the section to enter the Windows password/pin.

The issue is that the lock screen is shown, and the icons at the bottom right that are shown on that screen react to the mouse, they can be selected and show their option menus, indicating that the screen is not frozen and that it does react to user actions. But as you can see in the screenshot attached here, the problem is that the section to fill in the password/pin to unlock the screen is not shown anywhere, no matter how much I press the mouse all over the screen, for rather than send Ctrl+Alt+Delete keys or tell it to refresh the screen, nothing works, there is no way to unlock the screen by re-entering the password or pin, whatever I do it doesn't show up nowhere the text box to enter the password or the Windows pin.

I have tried Remmina, KRDC, Thincast and even launching xfreerdp from a terminal, but the same thing happens in all of them.

The version I'm using of FreeRDP is 3.0.0-dev (at least the one I see that uses Thincast) in a KDE Neon distro and I'm using X11 (not Wayland).

Attached below is a screenshot of the problem. As I say, the screen reacts and the buttons are active, but as you can see, the text box to enter the password/login is not shown anywhere.

Regards.

![Screenshot_20220414_234951](https://user-images.githubusercontent.com/103698556/163489760-5cace27b-9a5b-4d33-b58f-e715e831b8a2.png)

**Comment by akallabeth** [comment]
* most clients provide a alternate solution to send `ctrl+alt+del`, you tried these, right? `ctrl+alt+del` might be intercepted by your local system
* Does pressing some keyboard button help? (sometimes that is required to show the credential fields)
* Overall this looks like a server side thing to me, does it work with the microsoft client `mstsc` or does it have the same issue? 

closing until further information is provided pointing to a client side bug.

**Comment by Alden20** [comment]
Finally I have managed to find the cause of the problem, I put it here so that it can serve other users who may encounter the same issue.

Indeed, from another Windows 10 machine running mstsc the same problem occurred (so it is a Microsoft issue). After digging a bit in the configuration of the remote Windows 10 machine, I have found that the cause of the problem is that the remote machine uses Windows Hello as a validation system, and had an option active in the Sign-in options that says "Require Windows Hello sign-in for Microsoft accounts". If that option is active, the problem that I have mentioned occurs, after deactivating it, the section to enter the password to unlock the screen is already displayed.

Attached screenshot with the option that must be deactivated so that this problem does not occur:

![Screenshot_20220415_124221](https://user-images.githubusercontent.com/103698556/163562527-f12c2184-805d-4d5c-bae1-83ca3569e525.png)
